src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -118,6 +118,7 @@
       dataType: {
         dataType: 'rangedUUIDType',
       },
+      hidden: true,
       isIdColumn: false,
       queryable: true,
       essential: false,
```

Comment:
We might need to make this essential, but that can/should probably happen in a different ticket